src/modules/Remote.rb

```diff
@@ -131,54 +148,29 @@ def Propose
     # Read the current status
     # @return true on success
     def Read
-      # are the proper services enabled in xinetd?
-      xinetd_conf = SCR.Read(path(".etc.xinetd_conf.services")) || {}
-      xinetd_vnc1_enabled = xinetd_conf.any? { |c| c["service"] == "vnc1" && c["enabled"] }
-
-      log.info "VNC1 enabled: #{xinetd_vnc1_enabled}"
+      display_manager_remote_access = SCR.Read(path(".sysconfig.displaymanager.DISPLAYMANAGER_REMOTE_ACCESS")).to_s == "yes"
```

Comment:
you don't need ```to_s``` here

Reply:
Ok, I'll remove that. It was like that in `xinet_ready?`.

Reply:
The `to_s` is removed now.